test/manifold_test.cpp

```diff
@@ -586,6 +586,1038 @@ TEST(Manifold, PinchedVert) {
   EXPECT_EQ(touch.Genus(), 0);
 }
 
+TEST(Manifold, MeshWithCustomProps) {
+  MeshGL complex;
+  complex.numProp = 7;
+  complex.runOriginalID = {9};
+  complex.runIndex = {0, 144};
+
+  std::vector<float> vertProps;
+  vertProps = {0.0,
```

Comment:
You can combine these three lines into `complex.vertProperties = {0.0,`

Reply:
Can do when it's closer to completion.